.github/workflows/python-publish.yml

```diff
@@ -20,11 +20,11 @@ jobs:
     - name: Install dependencies
       run: |
         python -m pip install --upgrade pip
-        pip install setuptools wheel twine
+        pip install build twine
```

Comment:
I think this is the change that hung us up over on FLASC---presumably as long as thee `python -m build` line runs below, it should upload to PyPI correctly?

Reply:
That's right, I think I've gotten to learn from that and now this should go off fine, easy to test locally too but you don't want to run the twine call

Reply:
Yeah, ok, makes sense

Reply:
If you want to fully test it, you could [use the TestPyPI instance](https://packaging.python.org/en/latest/guides/using-testpypi/). If you did that, you could create a release on your fork and see that it uploads correctly without affecting anything on the primary repository.

Also heads up that this pull request is pointing to `main` rather than `develop`. Maybe that was intentional, but just pointing it out in case it wasn't.

Reply:
Should've been develop, moved over now